configs/tsup/package.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "@toss/tsup-config",
+  "version": "0.1.0",
+  "main": "./src/index.js",
+  "types": "./src/index.d.ts",
+  "devDependencies": {
+    "tsup": "^7.1.0"
+  },
+  "publishConfig": {
```

How about defining tsup v7 as peerDependency of this package

```suggestion
  "peerDependencies":{
    "tsup": "^7"
  },
  "publishConfig": {
```